artifacts

```diff
@@ -24,7 +24,11 @@ setup() {
 }
 
 prolong() {
-    echo "TODO"
+    echo ${NAME} | grep -qE '^[^/]+\:staging\-[0-9]{10}\.[0-9a-f]+\.[^./]+\.[0-9]+$'
```

Comment:
Crash an error if name is empty. Could be nice here, to ensure that people will use the input name.